core/drivers/crypto/versal/ecc.c

```diff
@@ -131,9 +131,9 @@ static TEE_Result ecc_prepare_msg(uint32_t algo, const uint8_t *msg,
 	if (msg_len > TEE_SHA512_HASH_SIZE + 2)
 		return TEE_ERROR_BAD_PARAMETERS;
 
-	if (algo == TEE_ALG_ECDSA_P384)
+	if (algo == TEE_ALG_ECDSA_SHA384 || algo == __OPTEE_ALG_ECDSA_P384)
```

Comment:
the `|| algo == __OPTEE_ALG_ECDSA_P384` part shouldn't be needed due to the call to `translate_compat_algo()` in `syscall_cryp_state_alloc()`, same below.

Reply:
OK, I will remove it. I suppose the same applies equally to "drivers: crypto: se050: do not use deprecated algorithm macros".